MapboxNavigation/Resources/Base.lproj/Navigation.storyboard

```diff
@@ -413,7 +407,7 @@
                                                 </userDefinedRuntimeAttributes>
                                             </view>
                                             <view hidden="YES" contentMode="center" translatesAutoresizingMaskIntoConstraints="NO" id="Q0R-gA-tRA" customClass="MBLaneArrowView">
-                                                <rect key="frame" x="240" y="0.0" width="30" height="30"/>
+                                                <rect key="frame" x="0.0" y="0.0" width="0.0" height="30"/>
```

Comment:
I didn't change anything here so it will probably change automatically again. It's not related to this PR but I'm going to revisit the layout of these lane views because their constraints break from time to time.